README.md

```diff
@@ -10,6 +10,8 @@ Quick start: `cargo run -- --config config.yml`
 
 This will run a proxy server with [config.yml](config.yml) as the configuration file.
 
+Run with `RUSTFLAGS="--cfg tokio_unstable"` to enable [tokio-console](https://github.com/tokio-rs/console)
```

Comment:
why it is not a feature?

Reply:
Because tokio_unstable is needed to enable experimental api so I thought with one flag we can turn on tokio console as well